SoftLayer/CLI/hardware/detail.py

```diff
@@ -61,6 +61,8 @@ def cli(env, identifier, passwords, price):
     table.add_row(['os_version', operating_system.get('version') or formatting.blank()])
     table.add_row(['created', result['provisionDate'] or formatting.blank()])
     table.add_row(['owner', owner or formatting.blank()])
+    table.add_row(['LastTransaction', result['lastTransaction']['transactionGroup']['name']])
```

Comment:
```suggestion
   table.add_row(['last_transaction', utils.lookup(result, 'lastTransaction', 'transactionGroup', 'name')])
```

Reply:
If were going to add the last transaction, I  think it would be helpful to know when it was run. So something like this:

```
last_transaction = "{} ({}}".format(utils.lookup(result, 'lastTransaction', 'transactionGroup', 'name'), 
                                                     utils.clean_time(utils.lookup(result, 'lastTransaction', 'modifyDate'))
table.add_row(['last_transaction', utils.lookup(result, 'lastTransaction', 'transactionGroup', 'name')])
```

That way we don't have to add an extra row in the table. Should end up looking like

`Service Setup (2021-01-01 12:15)`